.github/workflows/perfbench.yml

```diff
@@ -0,0 +1,37 @@
+name: Performance benchmark
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  benchmark:
+    name: Run performance benchmark
+    runs-on: ubuntu-latest
+    env:
+      PYTHONUNBUFFERED: 1
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install --upgrade pipenv
+          pipenv install --system
+
+      - name: Setup benchmark
+        run: |
+          scripts/perfbench/perfbench setup
+
+      - name: Run benchmark
+        env:
+          GITGUARDIAN_API_KEY: ${{ secrets.GITGUARDIAN_API_KEY }}
+        run: |
+          scripts/perfbench/perfbench run --times 3 ${{ github.event.inputs.run_options }}
```

Comment:
Suggestion : The 3 could be defined as an env var ?
```
env:
     NB_RUNS: 3
```

Reply:
What would we gain from this? Environment variables are easier to get wrong than CLI arguments: a typo in a CLI argument is reported by the tool, unlike a typo in an environment variable.

Reply:
What I mean was just making it explicit that the value 3 is a parameter and can be changed. Defining it as a variable at the top of the github workflow seems rather explicit.

Reply:
Ah I see :+1:. Just did it.